docs/src/conf.py

```diff
@@ -61,12 +62,12 @@
 # Configuration for intersphinx: refer to the Python standard library
 # and other packages
 intersphinx_mapping = {
-    "scipy": ("https://docs.scipy.org/doc/scipy/", None),
     "matplotlib": ("https://matplotlib.org/stable/", None),
     "numpy": ("https://numpy.org/doc/stable/", None),
+    "pandas": ("https://pandas.pydata.org/docs/", None),
```

Comment:
Does this allow a correct linking of external libraries in the in-code doc?

Reply:
Yes, but it should be done already. I just did the sorting of the librariers in alphabetical order to be consistent.